doc/manual/manual.tex

```diff
@@ -5,183 +5,241 @@
 \usepackage{hyperref}
 \usepackage{fullpage}
 
-\title{Arcade Learning Environment\\ Technical Manual}
+\title{Arcade Learning Environment\\ Technical Manual (v.0.5.0)}
 \author{}
 
 \begin{document}
 
 \maketitle
+
+\clearpage
+
 \tableofcontents
 
-\section{Overview}
+\clearpage
 
-This document describes how to install the Arcade Learning Environment as well as its technical 
-details, including 
-command-line flags, how to use the shared library interface, and the FIFO protocol 
-for agents communicating with ALE via stdin/stdout.
+\section{Overview}
 
-The following ways of interfacing with ALE are detailed in this document:
+This document describes how to install the Arcade Learning Environment as well as how to use it.
+We will present the different ways to implement an agent: 
+\begin{enumerate}
+  \item \textbf{Shared Library interface} (C++): Loads ALE as a shared library (Section 
+  \ref{sec:shared_library_interface}).}
```

Comment:
Bad section ref. I think we need to a \label{} to the shared library interface section.